test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/DotnetCliUtil.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace NuGet.XPlat.FuncTest
 {
-    internal class DotnetCliUtil
+    public class DotnetCliUtil
```

Comment:
This isn't available?
https://docs.microsoft.com/en-us/dotnet/core/api/system.reflection.assembly#System_Reflection_Assembly_Location